{
  "NServiceBus.MessageId": "b61ba0bc-300e-4769-9ae1-ac9000e7f1d2",
  "NServiceBus.MessageIntent": "Send",
  "NServiceBus.ConversationId": "9f599175-a5e8-43ca-b433-ac9000e7f1d4",
  "NServiceBus.CorrelationId": "b61ba0bc-300e-4769-9ae1-ac9000e7f1d2",
  "NServiceBus.ReplyToAddress": "ClientUI",
  "NServiceBus.OriginatingMachine": "DESKTOP-9PGLNM6",
  "NServiceBus.OriginatingEndpoint": "ClientUI",
  "$.diagnostics.originating.hostid": "38044fb5295632a834244a195e3c2d5d",
  "NServiceBus.ContentType": "text\/xml",
  "NServiceBus.EnclosedMessageTypes": "ConsoleApp1.message1, ConsoleApp1, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
  "NServiceBus.Version": "7.4.4",
  "NServiceBus.TimeSent": "2020-12-13 14:04:29:110362 Z",
  "NServiceBus.ExceptionInfo.ExceptionType": "NServiceBus.MessageDeserializationException",
  "NServiceBus.Retries.Timestamp": "2020-12-13 14:14:49:641090 Z",
  "NServiceBus.ExceptionInfo.InnerExceptionType": "System.Exception",
  "NServiceBus.ExceptionInfo.HelpLink": null,
  "NServiceBus.ExceptionInfo.Message": "An error occurred while attempting to extract logical messages from incoming physical message b61ba0bc-300e-4769-9ae1-ac9000e7f1d2",
  "NServiceBus.ExceptionInfo.Source": "NServiceBus.Core",
  "NServiceBus.ExceptionInfo.StackTrace": "NServiceBus.MessageDeserializationException: An error occurred while attempting to extract logical messages from incoming physical message b61ba0bc-300e-4769-9ae1-ac9000e7f1d2 ---> System.Exception: Could not determine type for node: 'message1'.\r\n   at NServiceBus.XmlDeserialization.InferNodeType(XmlNode node, Object parent) in \/_\/src\/NServiceBus.Core\/Serializers\/XML\/XmlDeserialization.cs:line 287\r\n   at NServiceBus.XmlDeserialization.Process(XmlNode node, Object parent, Type nodeType) in \/_\/src\/NServiceBus.Core\/Serializers\/XML\/XmlDeserialization.cs:line 202\r\n   at NServiceBus.XmlDeserialization.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in \/_\/src\/NServiceBus.Core\/Serializers\/XML\/XmlDeserialization.cs:line 56\r\n   at NServiceBus.XmlMessageSerializer.Deserialize(Stream stream, IList`1 messageTypesToDeserialize) in \/_\/src\/NServiceBus.Core\/Serializers\/XML\/XmlMessageSerializer.cs:line 62\r\n   at NServiceBus.DeserializeMessageConnector.Extract(IncomingMessage physicalMessage) in \/_\/src\/NServiceBus.Core\/Pipeline\/Incoming\/DeserializeMessageConnector.cs:line 119\r\n   at NServiceBus.DeserializeMessageConnector.ExtractWithExceptionHandling(IncomingMessage message) in \/_\/src\/NServiceBus.Core\/Pipeline\/Incoming\/DeserializeMessageConnector.cs:line 47\r\n   --- End of inner exception stack trace ---\r\n   at NServiceBus.DeserializeMessageConnector.ExtractWithExceptionHandling(IncomingMessage message) in \/_\/src\/NServiceBus.Core\/Pipeline\/Incoming\/DeserializeMessageConnector.cs:line 51\r\n   at NServiceBus.DeserializeMessageConnector.<Invoke>d__1.MoveNext() in \/_\/src\/NServiceBus.Core\/Pipeline\/Incoming\/DeserializeMessageConnector.cs:line 29\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in \/_\/src\/NServiceBus.Core\/Performance\/Statistics\/ProcessingStatisticsBehavior.cs:line 25\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at NServiceBus.TransportReceiveToPhysicalMessageConnector.<Invoke>d__1.MoveNext() in \/_\/src\/NServiceBus.Core\/Pipeline\/Incoming\/TransportReceiveToPhysicalMessageConnector.cs:line 37\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in \/_\/src\/NServiceBus.Core\/Pipeline\/MainPipelineExecutor.cs:line 45\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\r\n   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__10.MoveNext() in \/_\/src\/NServiceBus.Core\/Transports\/Learning\/LearningTransportMessagePump.cs:line 279",
  "NServiceBus.TimeOfFailure": "2020-12-13 18:46:21:386847 Z",
  "NServiceBus.ExceptionInfo.Data.Message ID": "b61ba0bc-300e-4769-9ae1-ac9000e7f1d2",
  "NServiceBus.ExceptionInfo.Data.Transport message ID": "88683868-e582-4307-bdee-ea65b0b1fce6",
  "NServiceBus.FailedQ": "ClientUI",
  "NServiceBus.ProcessingMachine": "DESKTOP-9PGLNM6",
  "NServiceBus.ProcessingEndpoint": "ClientUI",
  "$.diagnostics.hostid": "38044fb5295632a834244a195e3c2d5d",
  "$.diagnostics.hostdisplayname": "DESKTOP-9PGLNM6"
}